app/models/manageiq/providers/openstack/cloud_manager/scanning/job.rb

```diff
@@ -30,17 +30,16 @@ def call_snapshot_create
       options[:use_existing_snapshot] = false
       return unless create_snapshot
       signal(:snapshot_complete)
-    rescue => err
-      _log.log_backtrace(err)
-      signal(:abort, err.message, "error")
-      return
     rescue Timeout::Error
       msg = case options[:snapshot]
             when :smartProxy, :skipped then "Request to log snapshot user event with EMS timed out."
             else "Request to create snapshot timed out"
             end
       _log.error(msg)
       signal(:abort, msg, "error")
+    rescue => err
+      _log.log_backtrace(err)
+      signal(:abort, err.message, "error")
```

Comment:
This is a nice find...this is actually a bug fix (albeit a minor one).  In old versions, Timeout::Error did not derive from StandardError, but now it does.

Reply:
I wondered about that. I'm not sure when that changed.